chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,26 @@
+﻿$ErrorActionPreference = 'Stop'; # stop on all errors
+$toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$url        = 'https://github.com/jasongin/nvs/releases/download/v1.4.2/nvs-1.4.2.msi' # download url, HTTPS preferred
+# $url64      = '' # 64bit URL here (HTTPS preferred) or remove - if installer contains both (very rare), use $url

```

Comment:
Don't know if the installer is x86 or x64. In any case with how things are everything works so once you confirm I can probably delete these commented lines.

Reply:
The MSI has dual x86/x64 support. (Maybe very rare but I had some background with MSI so I knew how to do it.)